packages/project-access/src/project/i18n/write.ts

```diff
@@ -45,9 +46,10 @@ async function createUI5I18nEntriesBase(
     i18nPropertiesPaths: I18nPropertiesPaths,
     newEntries: NewI18nEntry[],
     modelKey: string,
-    fs?: Editor
+    fs?: Editor,
+    i18nFolder = ''
 ): Promise<boolean> {
-    const defaultPath = 'i18n/i18n.properties';
+    const defaultPath = join('i18n', i18nFolder, 'i18n.properties');
```

Comment:
This is also used as the value which is added to the manifest and we should not have windows delimiters there. I suspect this would cause issues when trying to load the resources at the runtime.

Reply:
Removed after call with @marufrasully